kensu/numpy/extractor.py

```diff
@@ -37,9 +37,14 @@ def extract_location(self, nd, location):
         if location is not None:
             return location
         else:
-            nd = self.skip_wr(nd)
-            # FIXME: or should it be added as prop to the ksu wrapper instead?!
-            return get_or_set_rand_location(nd)
+            if not isinstance(nd, kensu.numpy.ndarray):
+                # not wrapped numpy.ndarray do not (easily) allow to set extra attributes,
+                # so return a random uuid even if we expect a stable one
+                # otherwise - AttributeError: numpy.ndarray' object has no attribute '_ksu_loc_id'
+                print('WARN: kensu ndarraySupport.extract_location got unexpected arg type')
+                return get_rand_location()
+            else:
+                return get_or_set_rand_location(nd)
```

Comment:
P.S. can ndarray be used inside each cell of say pandas? if we have many many small ndarrays, we might want to switch to use even FASTER UUID for ndarray than the current MD5-based `uuid.uuid4()`